internal/backends/sqlite/collection.go

```diff
@@ -64,9 +64,22 @@ func (c *collection) Query(ctx context.Context, params *backends.QueryParams) (*
 		}, nil
 	}
 
-	q := fmt.Sprintf(`SELECT %s FROM %q`, metadata.DefaultColumn, meta.TableName)
+	var whereClaus string
+	var arg any
+
+	if params != nil && params.Filter != nil {
+		v, _ := params.Filter.Get("_id")
+		if v != nil {
+			if id, ok := v.(types.ObjectID); ok {
+				whereClaus = fmt.Sprintf(` WHERE %s = ?`, metadata.IDColumn)
+				arg = string(must.NotFail(sjson.MarshalSingleValue(id)))
+			}
+		}
+	}
+
+	q := fmt.Sprintf(`SELECT %s FROM %q`+whereClaus, metadata.DefaultColumn, meta.TableName)
 
-	rows, err := db.QueryContext(ctx, q)
+	rows, err := db.QueryContext(ctx, q, arg)
```

Comment:
Currently, if none of `args` are set, we pass `any` as the third param to `QueryContext`.
I think, a bit more accurate approach would be to make `arg` a slice and pass it as `arg...`. 
But if it works with `arg` being passed as is, it's not critical for now.

The comment is about the logic for here and below (explain).

Reply:
Updated, it's more correct to have array over empty string. :hugs: